sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/generators.scala

```diff
@@ -431,6 +432,22 @@ case class Explode(child: Expression) extends ExplodeBase {
     copy(child = newChild)
 }
 
+trait ExplodeGeneratorBuilderBase extends GeneratorBuilder {
```

Comment:
If we support the named argument for Explode, we should also update its ExpressionDescription and make sure `DESCRIBE FUNCTION` shows the correct information.